lib/list.gi

```diff
@@ -646,6 +646,18 @@ InstallOtherMethod(AsSSortedList,
      [ IsList ],
      l -> AsSSortedListList( AsPlist( l ) ) );
 
+InstallMethod( AsSSortedList,
+    "for a mutable strictly sorted list",
+    [ IsSSortedList and IsMutable ],
+    100
```

Comment:
Please don't use static rank adjustments like that except as a very, very last resort.

In this particular case, it would probably be fine to just use `SUM_FLAGS`. Alternatively, one could use a dynamic  rank adjustment via a rank function, which has the added benefit that if done right, it is "self-documenting", i.e., makes it clear which other filter you are trying to beat. But I think `SUM_FLAGS` is fine here.

Reply:
Will do! Thanks.

Reply:
Done. I also updated the commit message accordingly.